src/aspeak/api/api.py

```diff
@@ -42,16 +45,37 @@ def __init__(self, locale: Optional[str] = None, voice: Optional[str] = None,
             self._config.set_speech_synthesis_output_format(parse_format(audio_format))
         self._synthesizer = speechsdk.SpeechSynthesizer(self._config, self._output)
 
+    def config(self):
+        html = get(GET_TOKEN,verify=False)
```

Comment:
Unverified HTTPS request will trigger a warning.

I think there is no obvious reason to disable certificate verification.

```
/usr/lib/python3.10/site-packages/urllib3/connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'azure.microsoft.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
```

Reply:
Sorry, I forget to remove this. This is for debug.